src/domain/participant/index.js

```diff
@@ -76,55 +74,52 @@ const participantExists = (participant) => {
 
 const update = async (name, payload) => {
   try {
-    const participant = await Model.getByName(name)
+    const participant = await ParticipantModel.getByName(name)
     participantExists(participant)
-    await Model.update(participant, payload.is_disabled)
+    await ParticipantModel.update(participant, payload.isActive)
```

Comment:
It is done in order to return same response as in the GET participant by name. I agree it could be easily removed, but then that info won't be returned as result of the update. Please, confirm you're ok with that.